Update image names

sbesson opened the pull request:
This PR:

- migrates all the generation scripts under `scripts`
- refactors the script to loop over image types
- constructs the image name using the cell name rather than the original TIFF filename

sbesson commented:
@pwalczysko can you retry now from the tip of this branch. I have added some statements to help with the debugging and the overriding of the variable i.e. you should be able to run something along the lines of

```
BASE_DIRECTORY=/path/to/my/copy LIMIT=1 python scripts/make_filePaths_from_original_data.py 
```

pwalczysko commented:
What is actually ``root`` in the output above ?

pwalczysko commented:
As discussed, the LIMIT=-1 logic does not work. Proposing https://github.com/pwalczysko/idr0041-cai-mitoticatlas/commit/f9bfc146dc838715c3588ccf11b1dfd31856cc05 - we can now specify the START and STOP, where START has to be thought of as starting from 1 (first folder). 
For example

``START=2 STOP=3 BASE_DIRECTORY=/Users/pwalczysko/Desktop/mitotic_cell_atlas LIMIT=3 python scripts/make_filePaths_from_original_data.py``
will pick up the second and third folder.

''START=2 BASE_DIRECTORY=/Users/pwalczysko/Desktop/mitotic_cell_atlas LIMIT=3 python scripts/make_filePaths_from_original_data.py``
will pick the second, third and so on folders  all the way to the end.

I took the logic from https://stackoverflow.com/questions/36106712/how-can-i-limit-iterations-of-a-loop-in-python and it works fine for me locally. Note that some STEP could be included too, although this is not pressing.

joshmoore commented:
> What is actually root in the output above ?

The "root logger" as opposed to a specific "omero.blah" logger.

sbesson commented:
@pwalczysko pushed your change and refactored a bit the logging following your comments to be less verbose by default and more informative:
- increase default logging level is `INFO`
- allow debugging level to be controlled via envvar using the `DEBUG` envvar
- added a final statement about the number of files and folders detected

pwalczysko commented:
```
INFO:root:Deleting /Users/pwalczysko/Work/idr0041-cai-mitoticatlas/scripts/../experimentA/idr0041-experimentA-filePaths.tsv
INFO:root:Listed 120 files to import, located in 3 folders
```

Sorry for being a pain, but could there be:

    - an info about the script also creating the ...tsv file (now it looks like it only deletes one) - this is minor though
    - I would like to have in the output also the number of the top-folders in which the 3 folders are contained (especiallly in case I use START and STOP, I want to be sure the number of top folders is right, and this is an immediate check)

sbesson commented:
As discussed, fba3cc0 should add extra logging statements about the total number of assays folder detected as well as the number of assays folder used for generated the tsv file.

pwalczysko commented:
Functionality tested several times locally as well as remotely, varying the START and STOP values as well as the BASE_DIRECTORY value. 
The output works as expected and is readable.

The only open question here is the switching of the logging levels - not sure how to switch to DEBUG.

sbesson commented:
Thanks @pwalczysko. Should we merge this as this is getting pretty long and I open a PR to fix the debug leve? Can you also paste a screenshot showing the output of the image name logic?

pwalczysko commented:
![jk1](https://user-images.githubusercontent.com/2478303/37708434-90e4fe4e-2cfe-11e8-9495-be00b41e0628.png)


Yes, lets merge and follow-up. Is the screenshot okay ?

sbesson commented:
Thanks.
